Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -29,13 +29,26 @@ - (NSArray *)managedObjectContext:(NSManagedObjectContext *)context
         fetchSpan = [span startChildWithOperation:SENTRY_COREDATA_FETCH_OPERATION
                                       description:[self descriptionFromRequest:request]];
     }];
+
+    [SentryLog
```

Comment:
fetchSpan may be null, because scope span was null, which indicates we did not started a new span.

Same for `NSDataTracker`

Reply:
In NSDataTracker there is already a check for `if (ioSpan == nil) {`. I'll change this one.